README.md

```diff
@@ -1,296 +1,94 @@
-[![codecov](https://codecov.io/gh/zowe/vscode-extension-for-zowe/branch/master/graph/badge.svg)](https://codecov.io/gh/zowe/vscode-extension-for-zowe)
-[![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://slack.openmainframeproject.org/)
 
-[Zowe Explorer](https://github.com/zowe/community#zowe-explorer) is a sub-project of Zowe, focusing on modernizing mainframe experience. [Zowe](https://www.zowe.org/) is a project hosted by the [Open Mainframe Project](https://www.openmainframeproject.org/), a [Linux Foundation](https://www.linuxfoundation.org/) project.
+Welcome to Zowe Explorer! [Zowe](https://www.zowe.org/) is a project hosted by the [Open Mainframe Project](https://www.openmainframeproject.org/), a [Linux Foundation](https://www.linuxfoundation.org/) project.
 
-The Zowe Explorer extension modernizes the way developers and system administrators interact with z/OS mainframes by:
+Join our [Slack channel](https://slack.openmainframeproject.org/) to connect with the Zowe community.
 
-- Enabling you to create, modify, rename, copy, and upload data sets directly to a z/OS mainframe.
-- Enabling you to create, modify, rename, and upload USS files directly to a z/OS mainframe.
-- Providing a more streamlined way to access data sets, uss files and jobs.
-- Letting you create, edit, and delete Zowe CLI `zosmf` compatible profiles.
-- Letting you use the Secure Credential Store plug-in to store your credentials securely in the settings.
-- Letting you leverage the API Mediation Layer token-based authentication to access z/OSMF.
+## Requirements
 
-More information:
-
-- For the complete Zowe Explorer documentation, see [Zowe Docs](https://docs.zowe.org/stable/user-guide/ze-install.html).
-- Join the **#zowe-explorer** channel on [Slack](https://openmainframeproject.slack.com/) to stay in touch with the Zowe community.
-
-## Contents
-
-- [What's new in Zowe Explorer 1.12.0](#whats-new-in-zowe-explorer-1.12.0)
-- [Prerequisites](#prerequisites)
-- [Getting Started](#getting-started)
-- [Sample Use Cases](#sample-use-cases)
-- [Credentials Security](#credentials-security)
-- [Usage Tips](#usage-tips)
-- [Extending Zowe Explorer](#extending-zowe-explorer)
-
-## What's new in Zowe Explorer 1.12.0
-
-New features:
-
-- Added the ability to edit data set attributes before allocation
-- Allowed filtering of member names from the Data Sets search bar
-- Reorganized the context menus and streamlined the visible icons
-
-Bug Fixes:
-
-- Fixed the messaging displayed when handling inactive profiles and when updating profiles
-- Fixed the issue causing tree restructure when renaming a USS file or directory
-- Fixed the issue preventing issuing of commands when using profiles with tokens
-
-For more information, see [Changelog](https://marketplace.visualstudio.com/items/Zowe.vscode-extension-for-zowe/changelog).
-
-## Prerequisites
+Client-side prerequisites:
```

Comment:
I wouldn't add 'VS Code' as a prerequisite because this ReadMe is part of Marketplace. Seems redundant to me. Otherwise, I would list VS Code in the pre-req section

Reply:
@IgorCATech Some users may come to this page on GitHub outside of the VS Code context, so I think having it listed could be helpful in those cases. But you have a point, as well.